ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/tls/TlsConfiguration.scala

```diff
@@ -51,6 +52,20 @@ final case class TlsConfiguration(
           .build
       )
     else None
+
+  /** This is a side-effecting method. It modifies JVM TLS properties according to the TLS configuration. */
+  def setJvmTlsProperties(): Unit =
+    if (enabled && revocationChecks) enableOCSP()
+
+  /**
+    * Enables certificate revocation checks with OCSP.
+    * See: https://tersesystems.com/blog/2014/03/22/fixing-certificate-revocation/
+    */
+  private def enableOCSP(): Unit = {
+    System.setProperty("com.sun.net.ssl.checkRevocation", "true")
```

Comment:
Have you verified that this happens early enough that it's actually picked up by the relevant classes?

Reply:
Yes - I've verified this both by writing integration tests and manually by using a simple client which is connecting to the Sandbox with TLS.